### PR TITLE
Update Dockerfile and bump GLIBC_VERSION to 2.25

### PR DESCRIPTION
**Pull request by sam-caldwell**
Bumping glibc version to 2.25

### CODE REVIEW

**Comment by sam-caldwell** [comment]
sdurrheimer, since 2.25 is released, I thought I would bump the version here so the docker image will pass security scans complaining about vulnerabilities since 2.23.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM        alpine:3.2
 MAINTAINER  Steve Durrheimer <s.durrheimer@gmail.com>
 
-ENV GLIBC_VERSION="2.23-r1"
+ENV GLIBC_VERSION="2.25-r1"
```
Comment (sdurrheimer) [inline Dockerfile]: `2.25-r0` given that `2.25-r1` is still a pre-release.

**Review by sdurrheimer — CHANGES_REQUESTED** [review]
Thanks you !

Could you also change the github repo url given that it has been moved to `sgerrand/alpine-pkg-glibc`.

**Comment by sam-caldwell** [comment]
Change was submitted as requested.

**Review by sdurrheimer — APPROVED** [review]

**Comment by sdurrheimer** [comment]
Sorry for the delay.

**Pull request merged**